### PR TITLE
Mohamed chat messages front end create

### LINKED ISSUE #56: Mohamed chat messages front end create
Part of #5 Implements the frontend that allows someone to create a message

### PR DESCRIPTION
**Pull request by Mohamed-Elfouly**
Closes #56 Creates the frontend controls for creating/sending a message

### CODE REVIEW

**Review by cjyu81 — DISMISSED** [review]
Tests pass, looks good. Approving.

File: frontend/src/stories/components/ChatMessage/ChatDisplay.stories.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import { rest } from "msw";
+
+export default {
+  title: 'components/ChatDisplay',
+  component: ChatDisplay,
+};
+
+const Template = (args) => <ChatDisplay {...args} />;
+
+//const Template = () => <HelpRequestCreatePage storybook={true} />;
```
Comment (pconrad) [inline frontend/src/stories/components/ChatMessage/ChatDisplay.stories.js:15]: Please remove commented out code.
```suggestion
```

File: frontend/src/stories/components/ChatMessage/ChatPanel.stories.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import { rest } from "msw";
+
+export default {
+  title: 'components/ChatDisplay',
+  component: ChatDisplay,
+};
+
+const Template = (args) => <ChatDisplay {...args} />;
+
+//const Template = () => <HelpRequestCreatePage storybook={true} />;
```
Comment (pconrad) [inline frontend/src/stories/components/ChatMessage/ChatPanel.stories.js:15]: sigh.  You know what to do.
```suggestion
```

**Review by pconrad — CHANGES_REQUESTED** [review]
a couple of instances of commented out code. otherwise LGTM

**Comment by Mohamed-Elfouly** [comment]
Can't Resolve conflicts (button greyed out)

**Review by andrewthaii — APPROVED** [review]
LGTM

File: frontend/src/main/components/ChatMessage/ChatMessageForm.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import { Button, Form } from 'react-bootstrap';
+import { useForm } from 'react-hook-form'
+import { useNavigate } from 'react-router-dom';
+
+
```
Comment (pconrad) [inline frontend/src/main/components/ChatMessage/ChatMessageForm.js:6]:
```suggestion
```

**Pull request closed without merging**